app/views/pages/postcode.html.erb

```diff
@@ -0,0 +1,40 @@
+<body>
+<ul>
+  <% @stops.each do |stop| %>
+    <li>
+      <h2><%= stop["name"] %> </h2>
+      <table style="border-collapse: collapse; width: 100%;">
+        <thead>
+        <tr>
+          <th style="width: 20%;">Bus Number</th>
+          <th style="width: 30%;">Destination</th>
+          <th style="width: 50%;">Arrival Time (minutes)</th>
```

Comment:
Also good to move this to stylesheets and give the columns a class name - that way if you wanted to resize the headings you'd only update one of the widths

It might also be worth looking into flexboxes https://www.w3schools.com/css/css3_flexbox.asp which would automatically scale everything for you